shared/AppInsightsCommon/src/Util.ts

```diff
@@ -751,7 +751,7 @@ export class DateTimeUtils {
     /**
      * Get the number of milliseconds since 1970/01/01 in local timezone
      */
-    public static Now = CoreUtils.isUndefined(_window) ? () => new Date().getTime() :
+    public static Now = CoreUtils.isNullOrUndefined(_window) ? () => new Date().getTime() :
```

Comment:
getWindow() function returns null when window is not defined: https://github.com/microsoft/ApplicationInsights-JS/blob/7a800f19805716b137670299183cd4adbb8348ac/shared/AppInsightsCore/src/JavaScriptSDK/EnvUtils.ts#L30